openapi/src/parts/tps.yaml

```diff
@@ -258,4 +258,11 @@ components:
             $ref: '#/components/schemas/WsmPolicyLocation'
         regions:
           $ref: '#/components/schemas/Regions'
- 
+
+    WsmPolicySourceRequestBody:
```

Comment:
Since we are in the WSM API, I would expect this to always be the workspace id, not a generic `sourceObjectId`.

Since this body is only used by the merge check, I would expect it to live in the `workspace.yaml` and be called `MergeCheckRequest` in line with other schema objects.